src/localplayer.cpp

```diff
@@ -1038,13 +1040,20 @@ void LocalPlayer::old_move(f32 dtime, Environment *env, f32 pos_max_d,
 	/*
 		Check properties of the node on which the player is standing
 	*/
-	const ContentFeatures &f = nodemgr->get(map->getNodeNoEx(getStandingNodePos()));
+	const ContentFeatures &f_under = nodemgr->get(map->getNodeNoEx(
+		getStandingNodePos()));
+	const ContentFeatures &f_above = nodemgr->get(map->getNodeNoEx(
+		floatToInt(m_position + v3f(0, 0.2f * BS, 0), BS)));
+
 	// Determine if jumping is possible
-	m_can_jump = touching_ground && !in_liquid;
-	if (itemgroup_get(f.groups, "disable_jump"))
-		m_can_jump = false;
+	m_disable_jump = itemgroup_get(f_under.groups, "disable_jump");
+	m_can_jump = touching_ground && !m_disable_jump;
```

Comment:
Seems a possible change in behaviour in old_move (very dangerous), previously `m_can_jump` depended on `in_liquid` also.

Reply:
So did the new sneak code. Surely, I can undo these changes but in this case the `disable_ jump` in liquids will only work for people who use the new sneak code.

Reply:
Hmmm .. well, it's a minor feature, not much of a loss. I think it's best to not alter old_move.

Reply:
Ok, being able to jump in a liquid is probaly harmless, but needs testing.